libr/bin/p/bin_hunk.c

```diff
@@ -109,7 +100,6 @@ RBinPlugin r_bin_plugin_hunk = {
 	.check = &check,
 	.entries = &entries,
 	.sections = sections,
-	.symbols = &symbols,
```

Comment:
Uh?

Reply:
the bin hunk plugin doesnt handle any symbol stuff at all yet. i dont see reason to keep this "unused" line.